From 891abd23e923363720a78e74f7605c75259c5aae Mon Sep 17 00:00:00 2001
From: Matt Madison <matt@madison.systems>
Date: Sun, 15 Jan 2017 05:08:15 -0800
Subject: [PATCH 1/6] make.bash: fix target CC for build

For OE cross-builds, $CC_FOR_TARGET has more than one word
and needs to be quoted.

Upstream-Status: Pending
Signed-off-by: Matt Madison <matt@madison.systems>
---
 src/make.bash | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/make.bash b/src/make.bash
index 84aaab5..fcdbb52 100755
--- a/src/make.bash
+++ b/src/make.bash
@@ -172,7 +172,7 @@ if [ "$GOHOSTARCH" != "$GOARCH" -o "$GOHOSTOS" != "$GOOS" ]; then
 fi
 
 echo "##### Building packages and commands for $GOOS/$GOARCH."
-CC=$CC_FOR_TARGET "$GOTOOLDIR"/go_bootstrap install $GO_FLAGS -gcflags "$GO_GCFLAGS" -ldflags "$GO_LDFLAGS" -v std cmd
+CC="$CC_FOR_TARGET" "$GOTOOLDIR"/go_bootstrap install $GO_FLAGS -gcflags "$GO_GCFLAGS" -ldflags "$GO_LDFLAGS" -v std cmd
 echo
 
 rm -f "$GOTOOLDIR"/go_bootstrap
-- 
2.7.4

